minerl/Malmo/Schemas/MissionHandlers.xsd

```diff
@@ -3109,6 +3120,15 @@
         </xs:complexType>
     </xs:element>
 
+    <xs:element name="AgentQuitFromDeath">
+        <xs:annotation>
+            <xs:documentation>
```

Comment:
for agent to quit means to not respawn, correct?

Reply:
Correct - they will stop being in the episode and the number of active agents will decrease unless the server also wants to quit when any agent dies

Reply:
what happens if AgentQuitFromDeath is true, <ServerQuitFromDeath quitWhenAnyDead=false>, and all agents die? Would the server restart in that case?

Reply:
The episode should end when there are 0 agents and the episode has started. I will check